### ISSUE TITLE
Midterm Report Peer Review by Vidita Gawade

### ISSUE THREAD

**Issue by viditagawade11** [TSL-123/SentimentDrivenStrategy#12]
Project: Predict sentiment of news and using that prediction be able to predict the reaction in the financial world of trades. 

3 things I liked:
1. I really liked learning about the merge of finance and machine learning predictions, including what type of work has been already done.
2. I liked the extensive, detailed feature engineering placed in making sure words have different “weightage” if one negative word is more worse than a positive word, as well as learning about a dictionary created by Loughran and McDonald.
3. I also enjoyed learning about fuzzy match algorithm as well as Google n-gram
4. I think this idea, concept, and work is being done with great attention to detail.

3 things for improvement:
1. While it is nice of describing why you picked regularization over classification, and while I’m in this class so I can understand equations, I would advise for the final report to stray away from describing higher level equations that the sponsor of the project many not necessarily be diving too much into. If you do decide to keep it, which is also okay, be sure to describe in layman’s terms what the equations mean without having the reader figure out the equation.
2. I’m curious as to why ‘not’ is being counted as neutral, maybe I need more of an understanding, but isn’t not a negative word?
3. If I have to be picky: While your report shows great technical detail, for the final reporting to a business, it will be best to make sure usage of “we” versus “I” is consistent, and avoiding “don’t” - just type out do not. Also avoid saying statements like  “because of of the three page limit we have not explained it”.
4. Perhaps it was described but not communicated clearly, but, I’m unclear on how the price of an asset will be predicted… connect the price of an asset prediction better to the sentiment analysis. 

Overall, I can see that great in-depth work has been done, and in the future there’s exciting potential where the project is headed towards. I would just advice being more to the point and having more concise descriptions because I am having a hard time and am left curious as to how sentiment analysis will tie into helping to predict stocks well. But otherwise, I look forward to the results as its very interesting topic.

**Comment by muggle-magic** [comment]
Hi Vidita,

Thanks for your comments. 
For your questions about "how the price of an asset will be predicted"? We are not trying to predict asset price and asset price is not predictable. This is the reason why we are constructing a hedge trade here. We trade the company mentioned in the news and hedge the trade using S&P 500. So we don't care about the performance of this stock alone.  We only care about the performance of this particular stock comparing to the overall market. In other words, even if the particular stock drops, as long as it drops less than the overall market, we are in business.
For your question about why 'not' is being counted as neutral, consider the examples of 'not bad' and 'not good'. 'Not' is just a adverb, its sentiment is determined by the following word. This is why it is usually counted as neutral. Actually, in most of formal Natural Language Processing papers, expressions like 'not sth' are usually removed to avoid any confusion.

**Comment by viditagawade11** [comment]
Thank you for the quick response and clarification, that makes more sense.
